beaker/session.py

```diff
@@ -366,14 +458,106 @@ def invalidate(self):
 
     def load(self):
         "Loads the data from this session from persistent storage"
-        self.namespace = self.namespace_class(self.id,
-            data_dir=self.data_dir,
-            digest_filenames=False,
-            **self.namespace_args)
+        if self.old_reads():
```

Comment:
I guess we're ignoring the fact this might have a value here and another value later down the function, I think that's a risk we can take, it's very scoped, I think we should just be ready for one or two sentry errors.